packages/@icon-magic/cli/README.md

```diff
@@ -0,0 +1,121 @@
+# @icon-magic/cli
+
+> Icon Magic command line.
+
+## Install
+
+Using npm:
+
+```sh
+npm install --save-dev @icon-magic/cli
+```
+
+or using yarn:
+
+```sh
+yarn add @icon-magic/cli --dev
+```
+
+## Usage
+
+```
+Usage:
+  icon-magic [command] [options] <directories ...>
+
+Commands:
+  build               Construct flavors that an icon from its variants, after applying the build plugins.
```

Comment:
Might be good to briefly mention how the variants or flavors are provided (same for the other commands too...) — this will help hint about the configuration options for new users who are just looking at the usage docs

Reply:
I've added a glossary to the CLI and have linked to the other packages for more details.